<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="LAPTOP-5K9H8Q3R" failures="1" tests="10" name="Test1" time="0.152" errors="0" timestamp="2020-08-07T23:41:57 IST">
  <testcase classname="Demo1.Day1" name="Day" time="0.005">
    <failure type="java.lang.AssertionError" message="did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at Demo1.Day1.Day(Day1.java:14)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- Day -->
  <testcase classname="Demo1.Day3" name="toExclude" time="0.001"/>
  <testcase classname="Demo1.Day2" name="toInclude" time="0.009"/>
  <testcase classname="Demo1.Day3" name="toExclude3" time="0.025"/>
  <testcase classname="Demo1.Day2" name="catchData" time="0.001"/>
  <testcase classname="Demo1.Day3" name="toExclude4" time="0.001"/>
  <testcase classname="Demo1.Day2" name="catchData" time="0.001"/>
  <testcase classname="Demo1.Day2" name="catchData" time="0.001"/>
  <testcase classname="Demo1.Day2" name="toInclude5" time="0.008"/>
  <testcase classname="Demo1.Day2" name="Apanpahile" time="0.002"/>
  <testcase classname="Demo1.Day2" name="toInclude3" time="0.0">
    <ignored/>
  </testcase> <!-- toInclude3 -->
</testsuite> <!-- Test1 -->
